core/src/convert.rs

```diff
@@ -114,46 +113,88 @@ derive_into_ipld!(Map, BTreeMap<String, Ipld>, to_owned);
 derive_into_ipld!(Link, Cid, clone);
 derive_into_ipld!(Link, &Cid, to_owned);
 
+#[cfg(feature = "std")]
```

Comment:
You might be able to do something like (I haven't tried):

```rust
#[cfg(feature = "std")]
{
    derive_try_from_ipld!(Bool, bool);
    derive_try_from_ipld!(Integer, i8);
    …
}
```
which might make things easier to read.

Reply:
I tried you can't :(

Reply:
That sucks.